neqo-transport/src/connection.rs

```diff
@@ -1229,8 +1235,15 @@ impl Connection {
                 if ack_eliciting {
                     self.idle_timeout.on_packet_sent(now);
                 }
-                self.loss_recovery
-                    .on_packet_sent(space, hdr.pn, ack_eliciting, tokens, now);
+                self.loss_recovery.on_packet_sent(
```

Comment:
I think that this can be modified to take SentPacket here.  Clippy is right to complain about the number of arguments.  That means making SentPacket public, probably with a ::new constructor.

Reply:
hmm, it already is pub, so I think I'm on board with this.